tests/component-tests/textfieldControl/js/TextfieldStepperTests.js

```diff
@@ -148,17 +148,17 @@ https://github.com/fluid-project/infusion/raw/main/Infusion-LICENSE.txt
                         changeEvent: "{stepper}.applier.modelChanged"
                     }, {
                         // decrease to minimum
-                        jQueryTrigger: fluid.tests.textfieldControl.createKeyEvent("keydown", 40),
-                        element: "{stepper}.dom.textfield"
+                        jQueryTrigger: "click",
+                        element: "{stepper}.dom.decreaseButton"
```

Comment:
Did the keyboard interaction tests stop working after the change? It would be good to test if the up/down arrow keys still work to increment and decrement the model values.

Reply:
I reverted this change and now the tests hang locally (but on the TextFieldSlider tests, not this one). I'm going to see if they run in GitHub actions.

Reply:
Looks like they don't. No output, no messages.

Reply:
The "No output" is probably because the IoC testing framework is waiting for an event that never fires. In the browser you get an error in the console for this, but jqUnit is kind of stuck. I filed a [FLUID-6622](https://issues.fluidproject.org/browse/FLUID-6622) about this. 

Anyways, the error is probably because the browser is ignore the synthetic keyboard interaction. There are some interactions that the browser requires to be user directed to actually work. It seems that this might be one of those cases. In that case, we won't be able to test the keyboard interaction, and you can go back to using the mouse click on the buttons.

Thanks for looking into this though.

Reply:
That was my instinct based on a previous conversation we'd had which was why I removed it. I'll revert the revert.